File: ver_py/T06.py
# https://leetcode.cn/problems/powx-n/
class Solution:
    def myPow(self, x: float, n: int) -> float:
        return x**n


if __name__ == "__main__":
    x, n = map(float, input().split())
    print(Solution().myPow(x, n))

File: ver_cpp/T06.cpp
// https://leetcode.cn/problems/powx-n/
#include<bits/stdc++.h>
using namespace std;

class Solution {
public:
    double myPow(double x, int n) {
        return pow(x, n);
    }
};

int main() {
    double x;
    int n;
    cin >> x >> n;
    Solution s;
    cout << s.myPow(x, n) << endl;
    return 0;
}